testcases/crypto/aes_func.c

```diff
@@ -218,6 +223,12 @@ CK_RV do_EncryptDecryptUpdateAES(struct generated_test_suite_info * tsuite)
         goto testcase_cleanup;
     }
 
+    /* Skip tests if pkey = false, but the slot doesn't support CKM_AES_XTS*/
+    if (is_ep11_token(slot_id) && tsuite->mech.mechanism == CKM_AES_XTS && pkey == FALSE) {
+        testcase_skip("Slot doesn't support CKM_AES_XTS for non protected keys\n");
```

Comment:
I would write: "Slot supports AES-XTS only for protected keys.\n" Also at the other places.